skeleton/docker-entrypoint.sh

```diff
@@ -100,6 +100,11 @@ if [[ "$1" == "start" ]]; then
     $sudo $VENVBIN/zconsole run etc/${CONF} /app/scripts/create_site.py
   fi
   echo $MSG
+  if [ -n ${LISTEN_PORT} ] ; then
+    # Ensure the listen port can be set via container --environment.
+    # Necessary to run multiple backends in a single Podman / Kubernetes pod.
```

Comment:
What is the use case of running the same container twice in one pod? Generally, I would argue that it would make more sense to have multiple pods and scale them up/down with a deployment instead of scaling up the containers per pod.
The only reason for this I can come up with is if the proxy overhead is getting too high in a service mesh like istio.

Reply:
Additionally, I would suggest that if this was an option the `EXPOSE` in the Dockerfile be removed if this was merged, it will be confusing otherwise in my opinion.

Reply:
> What is the use case of running the same container twice in one pod?

Your whole deployment in a pod.  One ZEO, multiple backend containers, possibly 1-N frontend containers.  The sort of thing you would do pretty standard with `podman pod` in a docker-compose fashion.

@ericof Shouldn't the health check run a command against the running Plone server inside the container?  The health check now has access to the port via environment variable, so it should not be hard to modify it to continue working.